### ISSUE TITLE
代码中有两个loss相加的操作为什么要这样

### ISSUE THREAD

**Issue by Liufeiran123** [wntg/LLaMA-Omni#4]
lm_loss = llama_output.loss
  ctc_loss = self.speech_generator(llama_output['hidden_states'][-1], labels, tgt_units)
 loss = lm_loss + ctc_loss * self.config.ctc_loss_weight
代码中，两个loss相加，貌似是第一阶段的训练的。第一阶段对于speech_generator 会参与训练吗
